open_academy/i18n/es.po

```diff
@@ -0,0 +1,355 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* open_academy
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 15.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-08-11 17:11+0000\n"
+"PO-Revision-Date: 2022-08-11 17:11+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.report_session_template
+msgid "<strong>Attendee's name</strong>"
+msgstr "Nombre del asistente"
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.report_session_template
+msgid "<strong>End date:</strong>"
+msgstr "Fecha de finalizacion"
+
+#. module: open_academy
+#: model_terms:ir.ui.view,arch_db:open_academy.report_session_template
+msgid "<strong>Start date:</strong>"
+msgstr "Fecha de inicio"
```

Comment:
Like I said yesterday, you need to copy all the string's format

```suggestion
msgstr "<strong>Nombre del asistente</strong>"
```
Apply this to all this lines.

Reply:
Missing this